recaf-ui/build.gradle

```diff
@@ -0,0 +1,11 @@
+plugins {
+    id 'application'
+}
+
+dependencies {
+    implementation project(':recaf-core')
+}
+
+application {
+    mainClass = 'me.coley.recaf.RecafUI'
```

Comment:
We can also keep that simple.

```Groovy
jar {
    manifest {
        attributes('Main-Class': 'me.coley.recaf.RecafUI')
    }
}
```
No need for 3d-party plugins.

Reply:
I'm no gradle expert. But then if you try and run it in IntelliJ it doesn't actually run the app. It will compile, but not execute even though it is defined as an application launch config.

![image](https://user-images.githubusercontent.com/21371686/103221540-ea964680-48f0-11eb-996d-6bb48d37670b.png)

Reply:
NVM im just dumb